weaviate/gql/get.py

```diff
@@ -83,7 +84,12 @@ def __init__(
         if not isinstance(class_name, str):
             raise TypeError(f"class name must be of type str but was {type(class_name)}")
         if properties is None:
-            properties = []
```

Comment:
Can you adept the docstring?

Reply:
Done.

Reply:
that is not the docstring

Needs to be added here:
```
        Parameters
        ----------
        class_name : str
            Class name of the objects to interact with.
        properties : str or list of str
            Properties of the objects to interact with. <=== here
```

Reply:
Added the `schema` parameter to it at both the required places.

Reply:
I meant the properties entry, that needs an addition that `None` means "all properties"

Reply:
Also in query.get()